lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -578,6 +592,18 @@ export class PublicClientApplication implements IPublicClientApplication {
         return this.initializeAuthorizationRequest(request);
     }
 
+    /**
+     * Helper to validate app environment before making a silent request
```

Comment:
I guess this comment is incorrect as it validates silent and interactive requests.